packages/editor/vite.config.js.timestamp-1675717859489.mjs

```diff
@@ -0,0 +1,58 @@
+// ../../generateViteConfig.js
```

Comment:
fix: Please see if this is needed and remove if possible.

Reply:
removed